README.md

```
@@ -458,23 +270,7 @@ External package types (e.g. `uuid.UUID`) are resolved to primitives automatical
-router.POST("/users", func(c *gin.Context) {
-    var user CreateUserRequest
-    if err := c.ShouldBindJSON(&user); err != nil {
-        c.JSON(400, gin.H{"error": err.Error()})
-        return
-    }
-    c.JSON(201, user)
-})
-```
-
-The body and response types are analyzed even for anonymous handlers.
+Struct-level (cross-field) rules on a blank marker field (`_ struct{} \`validate:"gtefield=Min"\``) surface as a schema `description` note.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)